linux/LinuxProcessList.c

```diff
@@ -909,6 +909,24 @@ static void LinuxProcessList_readOomData(LinuxProcess* process, openat_arg_t pro
    fclose(file);
 }
 
+static void LinuxProcessList_readAutogroup(LinuxProcess* process, openat_arg_t procFd) {
```

Comment:
Maybe set `process->autogroup` and `process->autogroup_nice` to some invalid value first and print `N/A` in `LinuxProcess_writeField()` on those?